sys/stat.h is required for mode_t

Fixes
error: unknown type name 'mode_t'

--- plugins/sudoers/regress/env_match/check_env_pattern.c
+++ plugins/sudoers/regress/env_match/check_env_pattern.c
@@ -17,6 +17,7 @@
 #include <config.h>
 
 #include <stdio.h>
+#include <sys/stat.h>
 #include <stdlib.h>
 #ifdef HAVE_STRING_H
 # include <string.h>
